hackathon_site/dashboard/frontend/src/components/cart/CartCard/CartCard.module.scss

```diff
@@ -0,0 +1,39 @@
+@import "./../../../assets/abstracts/mixins";
+@import "./../../../assets/abstracts/variables";
+
+.Cart {
+    display: flex;
+    color: color(backgroundC);
+
+    &Pic {
+        height: 90px;
+        width: 90px;
+    }
+
+    &Name {
+        color: color(font);
+        padding-top: 11px;
+        padding-bottom: 17px;
+    }
+
+    &Parts {
+        display: flex;
+        flex-direction: column;
+        height: 90px;
+        width: 95%;
```

Comment:
Set width to 100%. On the Cart.js page, we'll set the width of the container the CartCard container is